ledger/eval_test.go

```diff
@@ -52,7 +52,7 @@ func TestBlockEvaluator(t *testing.T) {
 	defer l.Close()
 
 	newBlock := bookkeeping.MakeBlock(genesisInitState.Block.BlockHeader)
-	eval, err := l.StartEvaluator(newBlock.BlockHeader)
+	eval, err := l.StartEvaluator(newBlock.BlockHeader, 0)
```

Comment:
I'm curious why you chose to override the payset allocation inside StartEvaluator rather then when it was originally allocated in MakeBlock ( as nil, if I recall correctly ).